src/client/java/daos-java/src/main/native/io_daos_DaosClient.c

```diff
@@ -0,0 +1,207 @@
+/*
+ * (C) Copyright 2018-2020 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. B609815.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+
+#include "io_daos_DaosClient.h"
+#include <sys/stat.h>
+#include <gurt/common.h>
+#include <libgen.h>
+#include <stdio.h>
+#include <daos.h>
+#include <daos_jni_common.h>
+#include <fcntl.h>
+
+/**
+ * JNI method to open pool with given \a poolId.
+ *
+ * \param[in]	env		JNI environment
+ * \param[in]	clientClass	class of DaosFsClient
+ * \param[in]	poolId		pool UUID
+ * \param[in]	serverGroup	server group name
+ * \param[in]	ranks		ranks separated by ':'
+ *
+ * \return	copied pool handle in long
+ */
+JNIEXPORT jlong JNICALL
+Java_io_daos_DaosClient_daosOpenPool(JNIEnv *env,
```

Comment:
```suggestion
Java_io_daos_DaosClient_daosOpenPool(JNIEnv * env,
```

Reply:
@johannlombardi , The checkpath failed due to this request. It doesn't make sense. I'll get '(style) "foo * bar" should be "foo *bar"' if I follow this suggestion. please help check.